eng/pipelines/libraries/run-test-job.yml

```diff
@@ -33,11 +35,13 @@ jobs:
       pool: ${{ parameters.pool }}
       testScope: ${{ parameters.testScope }}
       ${{ if ne(parameters.liveRuntimeBuildConfig, '') }}:
-        displayName: ${{ format('Test Run {0} {1}', parameters.liveRuntimeBuildConfig, parameters.runtimeFlavor) }}
-        name: ${{ format('test_run_{0}_{1}', parameters.liveRuntimeBuildConfig, parameters.runtimeFlavor) }}
+        displayName: ${{ format('Test Run {0} {1}', parameters.liveRuntimeBuildConfig, parameters.runtimeDisplayName) }}
+        name: ${{ format('test_run_{0}_{1}', parameters.liveRuntimeBuildConfig, parameters.runtimeDisplayName) }}
       ${{ if eq(parameters.liveRuntimeBuildConfig, '') }}:
         displayName: 'Test Run'
         name: test_run
+      ${{ if eq(parameters.interpreter, 'true') }}:
+        testDisplayName: ${{ parameters.runtimeFlavor }}_interpreter_${{ parameters.liveRuntimeBuildConfig }}
```

Comment:
instead of adding a new parameter, we could use `runtimeDisplayName`. We could just flow `runtimeDisplayName` as a parameter to `base-job` and then declare `_testRunNamePrefixSuffix` to be : `runtimeFlavor_liveRuntimeBuildConfig` or `runtimeDisplayName_liveRuntimeBuildConfig` depending if `runtimeDisplayName` is empty or not.

Reply:
I think we need a new parameter in either case.  `runtimeDisplayName` does not exist in base_job.yml.  Because of that, I prefer `testDisplayName`, but I can be easily swayed ;-)

Reply:
Giving it a new thought, it seems better having this as we could have different test modes, like stress tests or what not.